[ews-app] Use passwords.json instead of environment variables

briannafan opened the pull request:
#### 81dce146552a6e7a7c62c263df83317527336394
<pre>
[ews-app] Use passwords.json instead of environment variables
<a href="https://bugs.webkit.org/show_bug.cgi?id=263331">https://bugs.webkit.org/show_bug.cgi?id=263331</a>
rdar://117155033

Reviewed by Aakash Jain.

Use util.load_password instead of os.getenv.

* Tools/CISupport/ews-app/ews-app/settings.py:
* Tools/CISupport/ews-app/ews/common/bugzilla.py:
(BugzillaBeautifulSoup.authenticate):
* Tools/CISupport/ews-app/ews/common/buildbot.py:
(Buildbot.retry_build):
* Tools/CISupport/ews-app/ews/config.py:
* Tools/CISupport/ews-app/ews/views/results.py:
(Results.post):

Canonical link: <a href="https://commits.webkit.org/269772@main">https://commits.webkit.org/269772@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/bdc54fc36d7121cf3cda564530e34e5c7236fb29

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/23471 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/1588 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/24595 "Built successfully") | [⏳ 🛠 wpe ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/21668 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/23741 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/3138 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/24013 "Built successfully") | [⏳ 🧪 wpe-wk2 ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/23714 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/1133 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/20310 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/26229 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/935 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/21208 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/27581 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/21389 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/21467 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/25245 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/918 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/18640 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/887 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/5614 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/1328 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/1193 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aj062 reviewed (approved):
Looks good. Thanks for making this change.

briannafan committed:
[ews-app] Use passwords.json instead of environment variables
https://bugs.webkit.org/show_bug.cgi?id=263331
rdar://117155033

Reviewed by Aakash Jain.

Use util.load_password instead of os.getenv.

* Tools/CISupport/ews-app/ews-app/settings.py:
* Tools/CISupport/ews-app/ews/common/bugzilla.py:
(BugzillaBeautifulSoup.authenticate):
* Tools/CISupport/ews-app/ews/common/buildbot.py:
(Buildbot.retry_build):
* Tools/CISupport/ews-app/ews/config.py:
* Tools/CISupport/ews-app/ews/views/results.py:
(Results.post):

Canonical link: https://commits.webkit.org/269772@main
